src/elm/Main.elm

```diff
@@ -1155,7 +1155,16 @@ update msg model =
             ( m, Cmd.none )
 
         VisibilityChanged visibility ->
-            ( { model | visibility = visibility, shift = False }, Cmd.none )
+            let
+                cmd =
+                    case visibility of
```

Comment:
if _returning_ to a tab (setting it to visible) refresh the page we are viewing!